test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -181,4 +181,22 @@ Describe "New-Item with links" -Tags @('CI', 'RequireAdminOnWindows') {
             $_.FullyQualifiedErrorId | Should Be "NewItemSymbolicLinkElevationRequired,Microsoft.PowerShell.Commands.NewItemCommand"
         }
     }
+
+    It "New-Item -ItemType SymbolicLink should understand directory path ending with slash" {
+        $folderName = [System.IO.Path]::GetRandomFileName()
+        try
+        {            
+            $symbolicLinkPath = New-Item -ItemType SymbolicLink -Path "$tmpDirectory/$folderName/" -Value "/bar/"
+            $symbolicLinkPath | Should Not Be $null
+        }
+        finally
+        {
+            $Directoy = Get-Item "$tmpDirectory/$folderName/" -ErrorAction SilentlyContinue
```

Comment:
This should probably be:

Get-Item "$tmpDirectory/$folderName/" -ErrorAction SilentlyContinue | remove-item -force -recurse -ErrorAction SilentlyContinue